index.js

```diff
@@ -40,24 +55,41 @@ module.exports = {
     function getBytesStored(feedId, cb) {
       /** @type {IndexPlugin} */
       const indexPlugin = ssb.db.getIndex('storageUsed')
-      ssb.db.onDrain('storageUsed', () => {
-        cb(null, indexPlugin.getBytesStored(feedId))
+      stats((err, info) => {
+        if (err) return cb(err)
+        ssb.db.onDrain('storageUsed', () => {
+          const logBytes = indexPlugin.bytesStored.get(feedId)
+          if (logBytes == null) return cb(null, 0)
+          cb(null, sumLogAndIndexes(logBytes, info))
+        })
       })
     }
 
     /**
      * @param {import('./types/helpers').CB<*>} cb
      */
     function stats(cb) {
-      ssb.db.onDrain('storageUsed', () => {
```

Comment:
A bit unrelated to this PR, but good to fix now: the `onDrain` is unnecessary here because the `getStats` is unrelated to the `storageUsed` index, it just uses FS APIs.